store/cache.go

```diff
@@ -10,16 +10,20 @@ import (
 // ErrDataExpired is returned when the data is expired.
 var ErrDataExpired = errors.New("data expired")
 
-// cache is an in-memory implementation of the KVStore. It is safe for concurrent read and write. The data stored will have
-// a valid duration. After the data expired, it will returns ErrDataExpired error to alert user to update the data.
+// ErrNoMoreItems is returned when no more items left in the iterator.
+var ErrNoMoreItems = errors.New("no more items in iterator")
+
+// cache is an in-memory implementation of the KVStore. It is safe for concurrent read and write. The data stored will
+// have a valid duration. After the data expired, it will returns ErrDataExpired error to alert user to update the data.
 type cache struct {
 	mu          *sync.RWMutex
 	data        map[string][]byte
 	lastSeen    map[string]int64
 	expiredTime int64
```

Comment:
The name `expiredTime` is confusing to me. In caching this is usually called `ttl` (or with a longer for being `timeToLive`).